### DISCUSSION TITLE
Editing Embeds

### DISCUSSION THREAD

**Discussion by Ansihv2008** [Rapptz/discord.py#7211] [Q&A]
How to edit embeds or messages. Pls send the code and the output code as an example. It will help me a lot. Thank u in advance

**Comment by pikaninja** [comment]
sending a message returns the message sent, for example:
```py
message = await channel.send("hi")
```
or, we can get the message by fetching it
```py
message = await channel.fetch_message(id) # you could also use channel.get_partial_message
```
then, we can use [message.edit](https://discordpy.readthedocs.io/en/latest/api.html#discord.Message.edit) to:
```py
await message.edit(content = "bye") # editing the content to "bye"
``` 
to edit embeds, pass the embed keyword argument to edit with a new embed

**Reply by Ansihv2008** [reply]
thx
